assets/ajax-cart.js.liquid

```diff
@@ -72,18 +78,23 @@ ShopifyAPI.onError = function(XMLHttpRequest, textStatus) {
     - Allow custom error callback
 ==============================================================================*/
 ShopifyAPI.addItemFromForm = function(form, callback, errorCallback) {
-  var params = {
+  var $body = $('body'),
```

Comment:
A bit of a nitpick, but can you change this to `$(document.body)` - it's slightly faster ([source](https://www.sitepoint.com/jquery-body-on-document-on/)).